### PR TITLE
PIM-11012 : Add helper if selected UI language is not sufficiently supported

### PR DESCRIPTION
**Pull request by jeremyBeaucousinAkeneo**
image

**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/UserManagement/Bundle/Resources/public/js/form/user-interface-helper-view.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {ThemeProvider} from 'styled-components';
+import {pimTheme} from 'akeneo-design-system';
+import {ReactView} from '@akeneo-pim-community/legacy-bridge/src/bridge/react';
+import {DependenciesProvider} from '@akeneo-pim-community/legacy-bridge';
+import {UserInterfaceHelper} from "./user-interface-helper";
+import {QueryClient, QueryClientProvider} from "react-query";
+
+class UserInterfaceHelperView extends ReactView {
```
Comment (pchasle) [inline src/Akeneo/UserManagement/Bundle/Resources/public/js/form/user-interface-helper-view.tsx:9]: I would suggest a more explicit naming in place of "User interface helper".
Something like "Unsupported ui locale warning".

```suggestion
class UnsupportedUiLocaleWarningView extends ReactView {
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# master
+__# master
```
Comment (amigaire) [inline CHANGELOG.md:1]: A missinput? 
```suggestion
# master
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -84,11 +84,12 @@
...
+- PIM-11012 : Add helper if selected UI language is not sufficiently supported
 
 ## Improvements
 
 - PIM-10782: Optimize get completeness SQL query
 
 ## New features
 
-## BC Breaks
+## BC Breaks__
```
Comment (amigaire) [inline CHANGELOG.md:95]: Same, missinput?

Comment (jeremyBeaucousinAkeneo) [reply CHANGELOG.md]: Thanks !

File: CHANGELOG.md (hunk 2 above)
Comment (jeremyBeaucousinAkeneo) [reply CHANGELOG.md]: Indeed, Thanks !

**Review by amigaire — APPROVED** [review]

**Pull request merged**